config/src/parameters/user.rs

```diff
@@ -179,8 +181,8 @@ impl Root {
                 emitter.emit(eyre!("\
                     The network consists from this one peer only (no `sumeragi.trusted_peers` provided). \
                     Since `--submit-genesis` is not set, there is no way to receive the genesis block. \
-                    Either provide the genesis by setting `--submit-genesis` argument, `genesis.private_key`, \
-                    and `genesis.file` configuration parameters, or increase the number of trusted peers in \
+                    Either provide the genesis by setting `--submit-genesis` argument, `genesis.file`, \
+                    `genesis.public_key` and `genesis.signature` configuration parameters, or increase the number of trusted peers in \
```

Comment:
- The only global change is renaming of `genesis.private_key` to `genesis.signature`
- `genesis.public_key` is a required parameter anyway, no need to mention it here IMO.

This will produce a diff minimal to the original:

```suggestion
                    Either provide the genesis by setting `--submit-genesis` argument, `genesis.signature`, \
                    and `genesis.file` configuration parameters, or increase the number of trusted peers in \
```